### PR TITLE
Native Image Feature.

### LINKED ISSUE #7061: GraalVM - Unable to load PublicSuffixDatabase.gz resource from the classpath
I'm using `com.squareup.okhttp3:okhttp:5.0.0-alpha.4` and `graalvm-ee-java11-22.0.0.2`
When using `native-image` I encounter this error:

```java
Exception in thread "OkHttp Dispjava.lang.IllegalStateException: Unable to load PublicSuffixDatabase.gz resource from the classpath.
        at okhttp3.internal.publicsuffix.PublicSuffixDatabase.findMatchingRule(PublicSuffixDatabase.kt:109)
        at okhttp3.internal.publicsuffix.PublicSuffixDatabase.getEffectiveTldPlusOne(PublicSuffixDatabase.kt:71)
```
My project is console kotlin app and there's no proguard or r8. I actually checked any that file `PublicSuffixDatabase.gz` is indeed inside my `jar` file but after creating native executable this happens.

### PR DESCRIPTION
**Pull request by yschimke**
Includes public suffix database, and test to confirm.

Closes #7061

### CODE REVIEW

File: native-image-tests/build.gradle.kts [hunk 1]
```diff
@@ -38,14 +39,12 @@ animalsniffer {
 }
 
 sourceSets {
-  // Not included in IDE as this confuses Intellij for obvious reasons.
   main {
     java.srcDirs(
-      "../okhttp/src/test/java",
```
Comment (yschimke) [inline native-image-tests/build.gradle.kts:44]: These tests have started to rely on internals which break this approach.  We should consider pulling out our tests into a test fixtures module or similar.  Less graal test coverage for now.

File: native-image-tests/build.gradle.kts [hunk 2]
```diff
@@ -60,11 +59,4 @@ graal {
   option("--allow-incomplete-classpath")
   option("--report-unsupported-elements-at-runtime")
   option("-H:+ReportExceptionStackTraces")
-
-  if (Os.isFamily(Os.FAMILY_WINDOWS)) {
```
Comment (yschimke) [inline native-image-tests/build.gradle.kts:64]: This has been fixed upstream.

**Comment by yschimke** [comment]
@swankjesse requesting post review.

**Review by swankjesse — COMMENTED** [review]
LGTM

Love this

**Pull request merged**